vcpkg/ports/qtpositioning/foregroundservice.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/src/plugins/position/android/src/jnipositioning.cpp b/src/plugins/position/android/src/jnipositioning.cpp
+index 2b2919f..bc42c2e 100644
+--- a/src/plugins/position/android/src/jnipositioning.cpp
++++ b/src/plugins/position/android/src/jnipositioning.cpp
```

Comment:
is this patch upstreamed already?

Reply:
No, I have to initiate a discussion upstream.